### ISSUE TITLE
Feedback week 4

### ISSUE THREAD

**Issue by lluisgomez** [idoiaruiz/mcv-m5#2]
I've been reading your article on Overleaf and I think it is well written. I have just a couple of comments on how to improve it. The section with the state of the art is too short, you must try to summarize the main ideas and techniques of the recent developments on each task: classification, detection, and segmentation (at the end of week 6). I miss some implementation details on sections 3.2 or 4.2: e.g. how did you trained the models? number of epochs, learning rate, training from scratch vs. using pre-trained weights, etc. Also, section 4.2 needs a bit more of critical appreciation of the results. Which was the best result?, why do you think it is that way? Draw some conclusions from tables 2 and 3. For this, you may also want to show some images with qualitative results.

As far as possible do not put plain URLs in the bibliography section. For datasets cite their original publication (if they have one). For code repositories it is better to add the URLs as a \footnote{} in latex.

I like the slides you have done for the object detection part. Maybe on slide 10 you should put the best results for each metric in bold text so the audience can easily catch the best method/variation. You can also add a slide with qualitative results if you think they will help on your presentation.

Regarding the integration of the SSD network on your code, I've seen it is almost done but not finished.  In any case I acknowledge you have done good work on this task. You still can add results of the SSD model on your final article and slides once we publish a solution for this task, ok? By the way, please, add credits at the begging of the following files: code/tools/ssd_utils.py and code/metrics/ssd_training.py.

Overall, I see all tasks for weeks 3/4 have been finished, except task (d) which is half done, and thus I guess you'll have a mark close to the maximum. Congratulations!
